lib/animina/narratives/resources/story.ex

```diff
@@ -1,6 +1,9 @@
 defmodule Animina.Narratives.Story do
+  alias Animina.Accounts
   alias Animina.Validations
 
+  @number_of_stories_required_for_complete_registration 1
```

Comment:
I am not a big fan of setting this value here. What when we need that number somewhere else too? e.g. in a template where we give the user some sort of feedback. It would be better to set it a default in `config.exs` with the possibility to override it for `prod` or `dev` environment.

Reply:
I have set t in config.exs